<launch>
  <param name="speed_limit" value="80.0" />

  <node name="speed_publisher" pkg="my_city" type="speed_publisher.py" />
  <node name="speed_subscriber" pkg="my_city" type="speed_subscriber.py" />
  
</launch>